src/api/spec/controllers/webui/request_controller/show_spec.rb

```diff
@@ -0,0 +1,438 @@
+require 'rails_helper'
+
+RSpec.describe Webui::RequestController, '#show', vcr: true do
+  let(:submitter) { create(:confirmed_user, :with_home, login: 'kugelblitz') }
+  let(:receiver) { create(:confirmed_user, :with_home, login: 'titan') }
+  let(:target_project) { receiver.home_project }
+  let(:target_package) { create(:package_with_file, name: 'goal', project_id: target_project.id) }
+  let(:source_project) { submitter.home_project }
+  let(:source_package) { create(:package, :as_submission_source, name: 'ball', project: source_project) }
+  let(:bs_request) do
+    create(:bs_request_with_submit_action,
+           description: 'Please take this',
+           creator: submitter,
+           target_package: target_package,
+           source_package: source_package)
+  end
+
+  describe 'GET #show' do
+    context 'when the user is part of the beta program' do
+      render_views
+
+      context 'when the user has only one submit action' do
+        let!(:comment) { create(:comment, commentable: bs_request) }
+
+        before do
+          Flipper.enable(:request_show_redesign, receiver)
```

Comment:
Repeating this in every context isn't needed if you put this in a `before` block of the parent context `'when the user is part of the beta program'`.

Reply:
It's true. Changing...